mvp-spec/publishers-requirements.md

```diff
@@ -173,21 +121,10 @@ data.preferences.source.signature
 ```
 <!--partial-end-->
 
-### Step 4: Send Transmission Requests with Inventory
+### Step 4: Send User Id and Preferences and Transmission Requests
```

Comment:
"and ... and" can be "and Preferences in Transmission Requests"

Reply:
Agreed